Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -34,6 +34,13 @@ open class NavigationMapView: UIView {
      */
     public var showsCongestionForAlternativeRoutes: Bool = false
     
+    /**
+     Controls wheter to show restricted portions of a route line.
+     
+     Restricted areas are drawn using `routeRestrictedAreaColor` which is customizable.
+     */
+    public var showsRestrictedAreasOnRoute: Bool = false
```

Comment:
Right now in CarPlay, we usually set the route line properties in the `CarPlayManagerDelegate.carPlayManager(_:didPresent:)` through `carPlayManager.carPlayNavigationViewController`. If we set the `showsRestrictedAreasOnRoute` here as `true`, and choose to start navigation from mobile, then the the route line maybe still don't show the restricted area. We may need to allow a setter of `showsRestrictedAreasOnRoute` to trigger the new route line generation and display when it enabled, otherwise it may not work in this certain case.

Reply:
You are right, restricted areas were not showing when connecting to CarPlay. I missed it because for tests I just changed the default value for this flag. Thanks for flagging!